### DISCUSSION TITLE
Callbag cookbook, Callbag data collections

### DISCUSSION THREAD

**Discussion by simplygreatwork** [callbag/callbag#69] [Show and tell]
Hello André and callbag community,

Thanks for creating this group, André.

I'd like to help create a cookbook of practical examples which use callbags. I've made several examples already: a reactive calculator like a spreadsheet, callbags for data collections and persistence, plus several other short examples. Another idea of a practical example might be logs filtering.

[Callbag Cookbook](https://github.com/simplygreatwork/callbag-cookbook)
[Callbag Collection](https://github.com/simplygreatwork/callbag-collection)

No big rush - but please me know if you have time to brainstorm ideas for more practical and more complete concrete examples.

**Comment by staltz** [comment]
A cookbook sounds like an excellent idea! People need more learning material such as that. I don't personally have time for this, but I support it happening. :)
